src/OrchardCore/OrchardCore.DisplayManagement/Html/CssOrchardHelper.cs

```diff
@@ -86,4 +87,38 @@ private static IEnumerable<string> Combine(string optionClasses, string[] additi
 
         return additionalClasses.Concat(new[] { optionClasses });
     }
+
+    private static IEnumerable<string> GetPartWrapperCssClasses(ContentTypePartDefinition partDefinition)
```

Comment:
This method should `return GetWrapperCssClasses("the", "part", "wrapper", "class", "names")`

Reply:
Mike please note, there are private and public methods, those public returns exactly what you want.

![image](https://github.com/OrchardCMS/OrchardCore/assets/6403130/eb337f4a-2934-49da-898f-59dabbac4324)